source/_integrations/v2c.markdown

```diff
@@ -47,8 +47,10 @@ The V2C integration currently exposes the following sensors:
 - charge_power: Current power charging your vehicle.
 - charge_energy: Energy transferred to your car during the current charging session.
 - charge_time: Time spent in the current charging session.
+- slave_error: Information from the EVSE on errors associated with the dynamic power control mode. Can be ignored when this mode is not used.
```

Comment:
looking at https://github.com/home-assistant/core/pull/118714, would this be *meter_error* now?

Reply:
Correct 👍

Reply:
```suggestion
- meter_error: Information from the EVSE on errors associated with the dynamic power control mode. It can be ignored when this mode is not used.
```